crates/lib/src/compiler/isa/qubit.rs

```diff
@@ -267,6 +274,50 @@ mod describe_rx_gates {
         ];
         assert_eq!(gates, expected);
     }
+
+    #[test]
+    fn it_defaults_to_0_for_missing_benchmark() {
+        let frb_sim_1q = FrbSim1q(Vec::new());
+        let gates = rx_gates(1, &frb_sim_1q).expect("should not error for missing benchmark");
+        let expected = vec![
+            Operator::Gate {
+                arguments: vec![Argument::Int(1)],
+                duration: 50.0,
+                fidelity: 1.0,
```

Comment:
`rx_gates` hard codes the fidelity value of 1.0 for `RX(0.0)`. I assume that's because it is a no-op.